docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,73 @@
+<p></p>
+#### 6. Did you add/remove a variant in the middle of an experiment?
+<p style="display:inline;">This is just a special case of #2.</p>
+<p></p>
+#### 7. Was sticky bucketing turned on and off in the middle of the experiment?
+<p style="display:inline;">Generally, once you start an experiment you should not change anything.</p>
+<p></p>
+#### 8. Are there a lot of users in the ["Individual Users Allocation"](../general/evaluation/implementation.md#Individual-inclusions) section?
+<p style="display:inline;">This can cause an SRM because you are not randomly allocating those users. It is fine to add some users in the experiment in the "Individual Users Allocation".</p>
+<p></p>
+#### 9. Is this your first experiment?
+<p style="display:inline;">Maybe there is an instrumentation bug. If you have had multiple SRMs, then the SRM is unlikely to be a false positive.</p>
+<p></p>
+#### 10. Are users logging in and out a lot?
+<p style="display:inline;">You may see this manifested in the variant jumping chart. Is the experiment causing users to log out more? This happens for a lot of financial institutions where you get logged out of your bank account if you are inactive for more than 15 minutes.</p>
+
+>    1. On log out regenerate device id<br>
+>    2. Amplitude thinks its a new user<br>
+>    3. User can get bucketed into a different variant<br>
+>    4. Then id gets resolved<br>
+
+This can also be related to anonymous users from sign up flow experiments. 
+<p></p>
+#### 11. Are there two users who have the same sequence of events but one is missing exposure?
+<p style="display:inline;">Maybe there are some cases where you are not sending the exposure event or are spending the exposure event when you should not be.</p>
+<p></p>
+#### 12. If the customer is not using the Amplitude SDK is the fall back variant causing issue?
+<p style="display:inline;">Is there some case where if the flag does not return a response in a reasonable amount of time, the flag default to control and counts that as an exposure? If so, control will have more exposures than treatment.</p>
+
+!!! info
+    Our client side sdk doesn’t send exposure events for fallback variant
+<p></p>
+#### 13. What is the [Platform user property](https://help.amplitude.com/hc/en-us/articles/215562387-User-property-definitions)?
+<p style="display:inline;">Are you sending data to Amplitude though <a href="https://www.docs.developers.amplitude.com/data/sources/segment/#javascript-client-side">Segment</a>, <a href="https://www.docs.developers.amplitude.com/analytics/what-is-amplitude/">Amplitude SDK</a>, <a href="https://www.docs.developers.amplitude.com/analytics/apis/batch-event-upload-api/">Batch API</a>, etc.</p><br>
+<p></p>
```

Comment:
Why is this important?

Reply:
If the data is going through Segment then it might not be our fault. if they are doing batch then they might have an event ordering issue.